book/src/future/groth-16.md

```diff
@@ -0,0 +1,60 @@
+# Groth16 Recursion
+Jolt's verifier today is expensive. We estimate 1-5 million gas to verify within the EVM. Better batching techniques on opening proofs can bring this down 5-10x, but it will remain expensive. Further, the short-term continuations plan (INESERT LINK) causes a linear blowup depending on the count of monolithic trace chunks proven.
+
+To solve these two issues we're aiming to add a configuration option to the Jolt prover with a post processing step, which creates a Groth16 proof of the Jolt verifier for constant proof size / cost (~280k gas on EVM) regardless of continuation chunk count or opening proof cost. This technique is industry standard.
+
+## Strategy
+The easiest way to understand the workload of the verifier circuit is to jump through the codebase starting at `vm/mod.rs Jolt::verify(...)`.  Verification can be split into 4 logical modules: [instruction lookups](https://jolt.a16zcrypto.com/how/instruction_lookups.html), [read-write memory](https://jolt.a16zcrypto.com/how/read_write_memory.html), [bytecode](https://jolt.a16zcrypto.com/how/bytecode.html), [r1cs](https://jolt.a16zcrypto.com/how/r1cs_constraints.html).
+
+Each of the modules do some combination of the following:
+- [Recursive sumcheck verification](https://jolt.a16zcrypto.com/background/sumcheck.html)
```

Comment:
wdym by recursive? If you mean this in the context of GKR, we do have non-GKR sumchecks too
```suggestion
- [Sumcheck verification](https://jolt.a16zcrypto.com/background/sumcheck.html)
```